applications/FluidDynamicsBiomedicalApplication/CMakeLists.txt

```diff
@@ -2,27 +2,26 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 message("**** configuring KratosFluidDynamicsBiomedicalApplication ****")
 
+kratos_add_dependency(${KRATOS_SOURCE_DIR}/applications/FluidDynamicsApplication)
+
 include_directories( ${CMAKE_SOURCE_DIR}/kratos )
 include_directories( ${CMAKE_SOURCE_DIR}/kratos/applications/FluidDynamicsApplication)
+include_directories( ${CMAKE_SOURCE_DIR}/kratos/applications/FluidDynamicsBiomedicalApplication)
```

Comment:
This is already the working this, no need for this.

Reply:
Sure.